Automattic/wp-calypso
# Experiment: Emphasize the refund policy on the plans step

#### Changes proposed in this Pull Request

This PR adds a new experiment to see how much the emphasized refund policy affects purchase conversions.
For more details, see pdgrnI-1H-p2

#### Testing instructions



* Create a website as a logged user or a logged-out user.
* You should be able to see the new copy emphasizing the 14-day money-back guarantee if you're assigned to the treatment of the `emphasizing_refund_policy` experiment.  
  <img width="1254" alt="Create_a_site_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/130585002-a214f9ab-fc2a-47bc-a91f-d0c7e38ed8c0.png">
* The banner should _NOT_ show up on the plans page of My site regardless of your experiment group.



cc @paulbonahora @southp @olaolusoga
